cerberus/errors.py

```diff
@@ -461,7 +461,7 @@ class BasicErrorHandler(BaseErrorHandler):
 
                 0x91: "one or more definitions validate",
                 0x92: "none or more than one rule validate",
-                0x93: "no definitions validate",
+                0x93: "not any definition validates",
```

Comment:
these have the same meaning, but the new one sounds more awkward

Reply:
i wanted to include a reference to the rule that evokes this message. do you have a proposal?